modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/search/engine/adapter/document/ElasticsearchDocumentRequestExecutorTest.java

```diff
@@ -42,21 +43,30 @@ public class ElasticsearchDocumentRequestExecutorTest {
 
 	@Before
 	public void setUp() throws Exception {
+		ElasticsearchDocumentRequestExecutor
+			elasticsearchDocumentRequestExecutor =
+				new ElasticsearchDocumentRequestExecutor();
+
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchDocumentRequestExecutor,
+			"_deleteByQueryDocumentRequestExecutor",
+			_deleteByQueryDocumentRequestExecutor);
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchDocumentRequestExecutor,
+			"_deleteDocumentRequestExecutor", _deleteDocumentRequestExecutor);
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchDocumentRequestExecutor,
+			"_indexDocumentRequestExecutor", _indexDocumentRequestExecutor);
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchDocumentRequestExecutor,
+			"_updateByQueryDocumentRequestExecutor",
+			_updateByQueryDocumentRequestExecutor);
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchDocumentRequestExecutor,
+			"_updateDocumentRequestExecutor", _updateDocumentRequestExecutor);
+
 		_elasticsearchDocumentRequestExecutor =
-			new ElasticsearchDocumentRequestExecutor() {
-				{
-					setDeleteByQueryDocumentRequestExecutor(
-						_deleteByQueryDocumentRequestExecutor);
-					setDeleteDocumentRequestExecutor(
-						_deleteDocumentRequestExecutor);
-					setIndexDocumentRequestExecutor(
-						_indexDocumentRequestExecutor);
-					setUpdateByQueryDocumentRequestExecutor(
-						_updateByQueryDocumentRequestExecutor);
-					setUpdateDocumentRequestExecutor(
-						_updateDocumentRequestExecutor);
-				}
-			};
+			elasticsearchDocumentRequestExecutor;
```

Comment:
@W2jx Please avoid this if possible, see my comments from the other PR.

Reply:
I will fix it.